### ISSUE TITLE
InsecureRequestWarning issue in empire_exec module kills module listing and use

### ISSUE THREAD

**Issue by 0x27** [byt3bl33d3r/CrackMapExec#120]
## Steps to reproduce

1. Install CrackMapExec as per instructions from Git
2. Try use any of the modules

## Command string used
``` 
$ sudo crackmapexec 192.168.10.112 -u testtest -p testtest -M mimikatz --verbose
```

## CME verbose output (using the --verbose flag)

```
Traceback (most recent call last):
  File "/usr/local/bin/crackmapexec", line 9, in <module>
    load_entry_point('crackmapexec==3.1.2', 'console_scripts', 'crackmapexec')()
  File "/usr/local/lib/python2.7/dist-packages/crackmapexec-3.1.2-py2.7.egg/cme/crackmapexec.py", line 190, in main
    modules = loader.get_modules()
  File "/usr/local/lib/python2.7/dist-packages/crackmapexec-3.1.2-py2.7.egg/cme/moduleloader.py", line 58, in get_modules
    m = self.load_module(os.path.join(path, module))
  File "/usr/local/lib/python2.7/dist-packages/crackmapexec-3.1.2-py2.7.egg/cme/moduleloader.py", line 43, in load_module
    module = imp.load_source('payload_module', module_path).CMEModule()
  File "/usr/local/lib/python2.7/dist-packages/crackmapexec-3.1.2-py2.7.egg/cme/modules/empire_agent_exec.py", line 6, in <module>
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: cannot import name InsecureRequestWarning
```

## OS
3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2 (2016-04-08) x86_64 GNU/Linux

## Target OS
Windows 10 (Build: 10586.26) x86_64

## Python Version
Python 2.7.9 (from Debian repos)

## Detailed issue explanation
TL;DR it coughs up that error every time I try use a "module". I was able to temporarily fix the issue by commenting out the following lines in `cme/modules/empire_agent_exec.py`:
```
from requests.packages.urllib3.exceptions import InsecureRequestWarning
requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```
This issue arises when trying to use *any* module, along with trying to list the modules with the -L flag. I believe it is due to a problem living deep in python-requests land for some reason.

**Comment by byt3bl33d3r** [comment]
@0x27 Huh, I'll spin up a Debian box and see if i can reproduce.  In the meantime try updating the requests package you're using, this might be happening because you're using an old version (e.g. ```pip install -U requests```)

**Comment by byt3bl33d3r** [comment]
@0x27 So I can't seem to reproduce this. I'm pretty sure it was because of an old version of the requests library. Did you manage to confirm this?

**Comment by byt3bl33d3r** [comment]
Closing since I can't reproduce this. Comment below if you still get this error.
